napari/components/tests/test_dims.py

```diff
@@ -104,6 +104,11 @@ def test_indices():
     assert dims.indices == (1, 2) + (slice(None, None, None),) * 2
 
 
+def test_axis_labels():
```

Comment:
this test is great, can you also add tests with the new `Dims` function signature

Reply:
I feel like this has been mentioned somewhere, already but haven't seen it in this PR... How about we make the default labels `[str(ax) for ax in range(ndim)]`? That will make it nicer to work with unlabeled axes, too!